src/modules/blockchain/implementation/ot-parachain/ot-parachain-service.js

```diff
@@ -0,0 +1,12 @@
+const Web3Service = require('../web3-service');
+
+class OtParachainService extends Web3Service {
+    constructor(ctx) {
```

Comment:
Add new method for gas price calculation, it will be different for each implementation.

For otp blockchain implement fetching of the gas price directly from the blockchain